src/main.cc

```diff
@@ -92,38 +139,20 @@ int main(int argc, char *argv[]) {
     const char *cmd_list = "Available commands: SendDeviceData, CheckUpdates, Download, Install, CampaignCheck, CampaignAccept, Pause, Resume, Abort";
     std::cout << cmd_list << std::endl;
 
-    std::vector<Uptane::Target> current_updates;
-    std::string buffer;
-    while (std::getline(std::cin, buffer)) {
-      std::vector<std::string> words;
-      boost::algorithm::split(words, buffer, boost::is_any_of("\t "), boost::token_compress_on);
-      std::string &command = words[0];
-      std::string &param = words[1];
-      boost::algorithm::to_lower(command);
-      if (command == "senddevicedata") {
-        aktualizr.SendDeviceData().get();
-      } else if (command == "checkupdates") {
-        auto result = aktualizr.CheckUpdates().get();
-        current_updates = result.updates;
-      } else if (command == "download") {
-        aktualizr.Download(current_updates).get();
-      } else if (command == "install") {
-        aktualizr.Install(current_updates).get();
-        current_updates.clear();
-      } else if (command == "campaigncheck") {
-        aktualizr.CampaignCheck().get();
-      } else if (command == "campaignaccept") {
-        aktualizr.CampaignControl(param, campaign::Cmd::Accept).get();
-      } else if (command == "pause") {
-        aktualizr.Pause();
-      } else if (command == "resume") {
-        aktualizr.Resume();
-      } else if (command == "abort") {
-        aktualizr.Abort();
-      } else if (!command.empty()) {
+    if (commandline_map.count("execute")) {
+      LOG_INFO << "REPL mode disabled";
```

Comment:
I would make this DEBUG level at most, since I suspect most users won't know what we're talking about.